pages/home.jsx

```diff
@@ -1,19 +1,131 @@
 var React = require('react');
 var Router = require('react-router');
 var Link = Router.Link;
+var moment = require('moment');
+var sanitizeHtml = require('sanitize-html');
 
 var HeroUnit = require('../components/hero-unit.jsx');
 var Blockquote = require('../components/blockquote.jsx');
 var Illustration = require('../components/illustration.jsx');
+var IconLinks = require('../components/icon-links.jsx');
+var IconLink = require('../components/icon-link.jsx');
+var IconButtons = require('../components/icon-buttons.jsx');
+var IconButton = require('../components/icon-button.jsx');
+
+var config = require('../lib/config');
+
+var blogFeedLoader = require('../lib/blog-feed-loader');
 
 var CaseStudies = React.createClass({
   render: function() {
     return (
-      <div className="case-studies">
-        <Blockquote className="primary-quote" author="Maurya C. New York, United States"
-            imgSrc="/img/pages/home/maurya-nyc.png" imgSrc2x="/img/pages/home/maurya-nyc@2x.png" imgAlt="Maurya NYC Quote">
-          <p>Web literacy is about more than coding - it's about how you can be a better web citizen.</p>
-        </Blockquote>
+      <div className="row">
+        <div className="col-sm-offset-2 col-sm-8 col-md-offset-2 col-md-8 col-lg-offset-2 col-lg-8">
+          <div>
+            <Blockquote author="Maurya C. New York, United States"
+                imgSrc="/img/pages/home/maurya-nyc.png" imgSrc2x="/img/pages/home/maurya-nyc@2x.png">
+              <p>"Web literacy is about more than coding - it's about how you can be a better web citizen."</p>
+            </Blockquote>
+          </div>
+        </div>
+      </div>
+    );
+  }
+});
+
+var FeaturedPost = React.createClass({
+  propTypes: {
+    data: React.PropTypes.object.isRequired
+  },
+  render: function() {
+    var parsedDate = moment(this.props.data.publishedDate);
+    return(
+      <div className="featured-post">
+        <div className="entry-posted-container">
+          <p className="entry-posted">
+            <time className="published" dateTime={this.props.data.publishedDate} >
+              <span className="posted-month">{parsedDate.format("MMM")}</span>
+              <span className="posted-date">{parsedDate.format("D")}</span>
+              <span className="posted-year">{parsedDate.format("YYYY")}</span>
+            </time>
+          </p>
+        </div>
+        <div className="entry-header-container">
+          <h3 className="entry-title"><a href={this.props.data.link}>{this.props.data.title}</a></h3>
+          <cite className="author">{this.props.data.author}</cite>
+        </div>
+        <p className="excerpt">
+          {this.props.data.contentSnippet}
```

Comment:
Hmm, since this is actual HTML and not text, any tags in the HTML will actually get escaped, so this will just look like raw HTML code to the user, rather than rendered HTML. I think you'll want to [dangerously set inner HTML](https://facebook.github.io/react/tips/dangerously-set-inner-html.html) here, but I recommend [avoiding HTML entirely for now](https://github.com/mozilla/teach.webmaker.org/pull/943/files#r31177174).